packages/shared/lib/core/wallet/actions/sendOutput.ts

```diff
@@ -15,6 +15,7 @@ export async function sendOutput(output: OutputTypes): Promise<void> {
         isTransferring.set(true)
         const account = get(selectedAccount)
         const transaction = await account.sendOutputs([output], DEFAULT_TRANSACTION_OPTIONS)
+        resetNewTransactionDetails()
```

Comment:
Why do we reset here? Maybe add an explanatory `NOTE: ...` comment or something.